easybuild/easyconfigs/s/SearchGUI/SearchGUI-3.3.3-Java-1.8.0_152.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PackedBinary'
+
+name = 'SearchGUI'
+version = '3.3.3'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://github.com/compomics/searchgui'
+description = """SearchGUI is a user-friendly open-source graphical user interface for configuring and 
+                 running proteomics identification search engines and de novo sequencing algorithms, 
+                 currently supporting X! Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda, OMSSA,
+                 Novor and DirecTag."""
```

Comment:
@korbini Better trim the whitespace on these lines to a single space, otherwise this will look quite strange in the output of `module show`